updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,166 @@
+    name: Extract cargo package name from Cargo.toml
+    kind: toml
+    scmid: "policy-repo"
+    dependson:
+      - condition#checkCargoTomlExists
+    spec:
+      file: "Cargo.toml"
+      key: "package.name"
+
+conditions:
+  checkCargoTomlExists:
+    name: Check if Cargo.toml exists
+    scmid: "policy-repo"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: Cargo.toml
+  checkCargoLockExists:
+    name: Check if Cargo.lock exists
+    scmid: "policy-repo"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: Cargo.lock
+
+targets:
+  updateMetadataAnnotationVersion:
+    name: update metadata.yml with new version
+    kind: yaml
+    scmid: "policy-repo"
+    sourceid: newestGitHubDraftReleaseVersion
+    spec:
+      searchpattern: true
+      file: "metadata.y*ml"
+      key: "$.annotations.'io.kubewarden.policy.version'"
+
+  updateCargoTomlVersion:
+    name: update Cargo.toml with new version
+    scmid: "policy-repo"
+    disablesourceinput: true
+    dependson:
+      - condition#checkCargoTomlExists
+    kind: shell
+    spec:
+      shell: /usr/bin/bash
+      command: |
+        export new_version='{{ source "newestGitHubDraftReleaseVersion" }}'
+        awk -v new_version="$new_version" '
+          BEGIN { in_package = 0 }
+          /^\[package\]/ { in_package = 1 }
+          in_package && /^version = / { sub(/version = ".*"/, "version = \"" new_version "\"") }
+          /^\[/ && !/^\[package\]/ { in_package = 0 }
+          { print }
+        ' Cargo.toml > Cargo.toml.tmp && mv Cargo.toml.tmp Cargo.toml
+      changedif:
+        kind: file/checksum
+        spec:
+          files:
+            - Cargo.toml
+
+  updateCargoLockVersion:
+    name: update Cargo.lock with new version
+    scmid: "policy-repo"
+    disablesourceinput: true
+    dependson:
+      - condition#checkCargoLockExists
+      - target#updateCargoTomlVersion
+    kind: shell
+    spec:
+      shell: /usr/bin/bash
+      command: |
+        export package_name='{{ source "cargoPackageName" }}'
+        export new_version='{{ source "newestGitHubDraftReleaseVersion" }}'
+        # note: if testing locally, copy the awk command without trailing spaces
+        awk -v new_version="$new_version" -v package_name="$package_name" '
+          BEGIN { package_found = 0 }
+          /^\[\[package\]\]/ { package_found = 0 }
+          $0 ~ "^name = \"" package_name "\"$" { package_found = 1 }
+          package_found && /^\s*version = / { sub(/version = ".*"/, "version = \"" new_version "\"") }
+          { print }
+        ' Cargo.lock > Cargo.lock.tmp && mv Cargo.lock.tmp Cargo.lock
+      changedif:
+        kind: file/checksum
+        spec:
+          files:
+            - Cargo.lock
+
+actions:
+  openUpdatePR:
+    title: 'build: Prepare for release {{ source "newestGitHubDraftReleaseVersion" }}'
+    kind: "github/pullrequest"
+    scmid: "policy-repo"
+    spec:
+      automerge: {{ .pr.automerge }}
```

Comment:
For info it's not possible since v0.96.0 to specify reviewers

cfr https://www.updatecli.io/docs/plugins/actions/github/

Reply:
I would leave the reviewers unset, they are already set by the Terraform automation and allows for the subsequent reusable workflow to be simpler.

Reply:
Done in https://github.com/kubewarden/automation/pull/99/commits/8522fc3fada276b17452f7c81be6e70aa2312eb1.